### ISSUE TITLE
macOS hardware acceleration is broken

### ISSUE THREAD

**Issue by adill** [electron/electron#19631]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 7.0.0-beta.1
* **Operating System:**
  * macOS 10.14.6
* **Last Known Working Electron version:**
  * 6.0.0

### Expected Behavior
macOS rendering should look the same whether hardware acceleration is enabled or disabled.

### Actual Behavior
macOS rendering exhibits serious artifacts (white squares) when hardware acceleration is enabled.

### To Reproduce

```sh
$ git clone https://github.com/adill/electron-quick-start/ -b macOS-hardware-accel
$ npm install
$ npm start || electron .
```

- Click the 'Electron' link to load the Electron repo on GitHub
- Move your mouse in a circle around the window
- Observe the artifacts

**Comment by nornagon** [comment]
Appears to be fixed in 8.0.0-nightly.20190803, so i suspect this is a Chromium issue and will be fixed in the next roll. Let's make sure we stay on top of it though!

**Comment by codebytere** [comment]
Fixed in latest release of v7.
